### DISCUSSION TITLE
Helm Release for Jaeger V2

### DISCUSSION THREAD

**Discussion by grdq** [jaegertracing/jaeger#6212] [Q&A] [answered]
Hi,

I'm very excited at the news that Jaeger V2 has a release now! I was just about to deploy a V1 Jaeger cluster but wondering if it is worth waiting for the release of the [new charts](https://github.com/jaegertracing/helm-charts/issues/610) that support Jaeger V2. Is this around the corner given V2 is out, or is there further work for V2 to become GA before there is a helm release?

I don't mind either way, just want to know if to just deploy the cluster running V1 or to wait a week or so 😄 

Thanks!

**Comment by jkowall** [comment, marked as answer]
As you pointed out we have an active project through our LFX Mentorship program this semester on building out both the Helm Chart and the Operator for Jaeger v2. Jaeger v2 is Generally Available already, but the Chart and Operator will come within the next month (hopefully). 

Remember when Jaeger v1 was first released there was no Helm or Operator Framework as those projects did not exist. You can still start testing with the current distribution and even start to process production traces with it to ensure you understand your config setup and roles properly due to the changes in the way V2 is configured.

**Reply by grdq** [reply]
Sounds good, thanks for the info
